src/nanoemoji/svg.py

```diff
@@ -405,14 +415,15 @@ def _picosvg_docs(
         reuse_cache = ReuseCache()
         # establish base svg, defs
         svg = SVG.fromstring(
-            r'<svg version="1.1" xmlns="http://www.w3.org/2000/svg"><defs/></svg>'
-        )
+            r'<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs/></svg>'
```

Comment:
don't manually add the xlink ns, it's added automatically depending on the usage of xlink attributes when you do `tostring`. Just add the xlink:href to the use element, and we're done

```suggestion
            r'<svg version="1.1" xmlns="http://www.w3.org/2000/svg"><defs/></svg>'
```

Reply:
OK, removed.